app/models/manageiq/providers/autosde/inventory/collector/storage_manager.rb

```diff
@@ -73,4 +73,17 @@ def physical_storage_families
       }
     end
   end
+
+  def storage_hosts_wwpn_candidates
+    @storage_hosts_wwpn_candidates ||= @manager.autosde_client.StorageHostWWPNCandidatesApi
+                                               .storage_hosts_wwpn_candidates_get.map do |candidate|
+      {
+        :candidates => candidate.wwpn,
+        :physical_storage_id => candidate.system_uuid,
```

Comment:
I think you want to do a `persister.physical_storages.lazy_find(candidate.system_uuid)` here.  Weird that it is in the collector since this is typically done in the parser.